index.html

```diff
@@ -0,0 +1,170 @@
+                <img src="./images/icons/github.png" alt="github" />
+              </a>
+            </li>
+            <li>
+              <a href="#">
+                <img src="./images/icons/angellist.png" alt="angellist" />
+              </a>
+            </li>
+          </ul>
+        </div>
+        <div class="scroll-down-icon">
+          <a href="#">
+            <img
+              src="./images/icons/Disabled.png"
+              class="fa"
+              alt="scroll-down"
+            />
+          </a>
+        </div>
+      </section>
+
+      <section id="Portfolio" class="works-section d-flex">
+        <div class="works-section-title">
+          <h2 class="h2-size">My Recent Works</h2>
+        </div>
+        <ul class="card-container"></ul>
+      </section>
+
+      <article class=""></article>
+
+      <section class="about-section" id="About">
+        <div class="about d-flex flex-center">
+          <h2 class="h2-size">About me</h2>
+          <p>
+            I can help you build a product , feature or website Look through
+            some of my work and experience! If you like what you see and have a
+            project you need coded, don’t hestiate to contact me.
+          </p>
+          <button type="button" class="btn-resume">Get My Resume</button>
+        </div>
+        <ul class="card-wrapper flex-center">
+          <li class="cards d-flex flex-center">
+            <img
+              src="./images/icons/icon-languages@2x.svg"
+              class="tools-img"
+              alt="languages"
+            />
+            <h3>Languages</h3>
+            <ul class="languages d-flex">
+              <li>JavaScript</li>
+              <li>Ruby</li>
+              <li>Html</li>
+              <li>CSS</li>
+            </ul>
+          </li>
+          <li class="cards d-flex flex-center">
+            <img
+              src="./images/icons/icon-frameworks@2x.svg"
+              class="tools-img"
+              alt="frameworks"
+            />
+            <h3>Frameworks</h3>
+            <ul class="languages d-flex">
+              <li>Bootstrap</li>
+              <li>React & Redux</li>
+              <li>SASS</li>
+              <li>Ruby on rails</li>
+              <li>PostgreSQL</li>
+            </ul>
+          </li>
+          <li class="cards d-flex flex-center">
+            <img
+              src="./images/icons/icon-skills@2x.svg"
+              class="tools-img"
+              alt="skills"
+            />
+            <h3>Skills</h3>
+            <ul class="languages d-flex">
+              <li>GitHub</li>
+              <li>Jest</li>
+              <li>Heroku</li>
+              <li>TDD</li>
+              <li>RSpec</li>
+              <li>Dev Tools</li>
+              <li>Capybara</li>
+            </ul>
+          </li>
+        </ul>
+      </section>
+      
+      <footer class="footer">
+        <div class="mail-container">
+          <a href="#">theoenste99@gmail.com</a>
```

Comment:
[OPTIONAL]

Another great way for users to reach you is via e-mail, maybe you can add the option to mail you on click with an: `href="mailto:theoenste99@gmail.com"` :1st_place_medal: